RunningOrder/Story/StoryList.swift

```diff
@@ -8,17 +8,28 @@
 
 import SwiftUI
 
-extension Story: Identifiable {
-    var id: String { ticketReference }
-}
+extension Story: Identifiable {}
 
 struct StoryList: View {
-    let sprintIndex: Int
 
-    var sprint: Sprint { return sprintManager.sprints[sprintIndex] }
+    let sprint: Sprint
 
     @EnvironmentObject var toolbarManager: ToolbarManager
-    @EnvironmentObject var sprintManager: SprintManager
+
+    @EnvironmentObject var storyManager: StoryManager
+    @EnvironmentObject var storyInformationManager: StoryInformationManager
+
+    var createdStoryBinding: Binding<Story?> {
+        return Binding<Story?>(
+            get: { return nil },
+            set: { newValue in
+                if let story = newValue, var currentStories = storyManager.stories[sprint.id] {
+                    currentStories.append(story)
```

Comment:
maybe a helper inside the manager could help : `storyManager.add(story, toSprint: sprint.id)`